client/src/plugins/version-info/ReleaseInfo.js

```diff
@@ -44,35 +44,39 @@ export function ReleaseInfo(props) {
-          Element templates will now work for Camunda 8 BPMN diagrams, too.
-          We will now also use a more concise naming: <i>Camunda Platform</i> becomes Camunda 7
-          and <i>Camunda Cloud</i> becomes Camunda 8.<br />
-          Learn more about Camunda 8 in general <a href="https://camunda.com/blog/2022/04/camunda-platform-8-orchestrate-all-the-things?utm_source=modeler&utm_medium=referral">in the Camunda 8 blogpost.</a>
+          <h4>Context pad for multi-selection</h4>
+          This release adds the context pad to multiple selected elements.<br />
+          You can now conveniently perform actions on multiple elements, such as Align and Distribute.
+          We also improved the behavior of this actions to be more intuitive and to give you more control.
         </li>
         <li>
-          <h4>Improved overall UI to become a "Visual IDE"</h4>
-          We re-worked the entire Modeler, including welcome screen, properties panel, status bar, tab bar, and bottom panel
-          following our visions of a "Visual IDE".<br />
-          Learn more <a href="https://camunda.com/blog/2022/01/camunda-modeler-5-0-0-alpha-0-released?utm_source=modeler&utm_medium=referral">in a dedicated blog post about the UI improvements</a>.
+          <h4>Refined selection UX</h4>
+          We refined the whole selection behavior to make changes to complex process models more convenient.<br />
+          Clear outlines now indicate what the outer box and child elements of a selection are.
+          In addition, the properties panel is now consistent with the selection and allows changes only
+          when a single element is purposefully selected.
         </li>
         <li>
-          <h4>Process Debugging for Camunda 8 BPMN diagrams</h4>
-          The Modeler will now support you when automating BPMN processes with Camunda 8 by showing implementation errors,
-          and giving hints on how to fix them.
+          <h4>Conditional properties for element templates</h4>
+          This release adds support for conditional properties in your element templates.<br />
+          You can now create dynamic element templates that make the properties panel respond to user input
+          by showing or hiding fields depending on the input of other fields.<br />
+          You can learn more about how to define conditional properties <a href="https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/defining-templates/?utm_source=modeler&utm_medium=referral">in the element templates documentation</a>.
         </li>
         <li>
-          <h4>Support drill down into collapsed subprocesses</h4>
-          In Camunda 7 BPMN diagrams, you can now model collapsed subprocesses by
-          collapsing an expanded subprocess. Use the blue drill down icon to view and model the contents of the subprocess.
+          <h4>Adjusted telemetry</h4>
```

Comment:
(I personally would not mention this).

"We added additional data points to our telemetry to better understand how our app works. As always, you decide whether you want that information to be shared. Find a comprehensive overview of what we track, how we do this and why [here]."

Reply:
(if so as the last point on the list).

Reply:
Thanks; I updated accordingly